pkg/cluster/operation/check.go

```diff
@@ -58,6 +58,7 @@ var (
 	CheckNameSELinux     = "selinux"
 	CheckNameCommand     = "command"
 	CheckNameFio         = "fio"
+	CheckNameTHP         = "THP"
```

Comment:
To keep up with the previous code style, use lowercase

Reply:
thx, got it